cdk/domino/domino_stack.py

```diff
@@ -323,7 +342,26 @@ def provision_eks_cluster(self):
                     lts = eks.LaunchTemplateSpec(id=lt.launch_template_id, version=lt.version_number)
                     disk_size = None
 
-                ng = self.cluster.add_nodegroup_capacity(
+            managed_policies = [
+                self.ecr_policy,
+                self.s3_policy,
+                self.autoscaler_policy,
+                iam.ManagedPolicy.from_aws_managed_policy_name('AmazonEKSWorkerNodePolicy'),
+                iam.ManagedPolicy.from_aws_managed_policy_name('AmazonEC2ContainerRegistryReadOnly'),
+                iam.ManagedPolicy.from_aws_managed_policy_name('AmazonEKS_CNI_Policy'),
+            ]
+            if self.config["route53"]["zone_ids"]:
+                managed_policies.append(self.route53_policy)
+
+            ng_role = iam.Role(
```

Comment:
If we're going to define the role from scratch rather than let EKS create it, maybe it'd make sense to just make a single role and assign that to all the managed/unmanaged nodegroups?

Reply:
I was thinking about it.

I leave it to the case when managed and unmanaged nodegroup actually has some differences, including the roles.

Perhaps one role for all managed and one role for all unmanaged. I'll do it as a separate change

So far I define a single policy and attack it to the both roles.